src/Features/CSharp/Portable/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -114,14 +115,23 @@ private static async Task HandleSingleTypeAsync(CompletionContext context, Seman
                 type = typeArg;
             }
 
+            var showTypeOnlyIfMembers = false;
+            INamedTypeSymbol? lazyEnclosingNamedType = null;
```

Comment:
this was super confusing to me.  i couldn't see how this value was being written before being read.  i'm not a fan of doing tihs and then having a local function setting this.

Reply:
i don't see value in this being lazy.  it's not expensive to compute, and tons of other services will have already asked for this.

Reply:
⏱️ Making eagerly computed